docs/data/material/pages.ts

```diff
@@ -0,0 +1,240 @@
+          { pathname: '/material/react-lists', title: 'List' },
+          { pathname: '/material/react-tables', title: 'Table' },
+          { pathname: '/material/react-tooltips', title: 'Tooltip' },
+          { pathname: '/material/react-typography' },
+        ],
+      },
+      {
+        pathname: '/material/components/feedback',
+        subheader: 'feedback',
+        children: [
+          { pathname: '/material/react-alert' },
+          { pathname: '/material/react-backdrop' },
+          { pathname: '/material/react-dialogs' },
+          { pathname: '/material/react-progress' },
+          { pathname: '/material/react-skeleton' },
+          { pathname: '/material/react-snackbars', title: 'Snackbar' },
+        ],
+      },
+      {
+        pathname: '/material/components/surfaces',
+        subheader: 'surfaces',
+        children: [
+          { pathname: '/material/react-accordion' },
+          { pathname: '/material/react-app-bar' },
+          { pathname: '/material/react-cards', title: 'Card' },
+          { pathname: '/material/react-paper' },
+        ],
+      },
+      {
+        pathname: '/material/components/navigation',
+        subheader: 'navigation',
+        children: [
+          { pathname: '/material/react-bottom-navigation' },
+          { pathname: '/material/react-breadcrumbs' },
+          { pathname: '/material/react-drawers', title: 'Drawer' },
+          { pathname: '/material/react-links', title: 'Link' },
+          { pathname: '/material/react-menus', title: 'Menu' },
+          { pathname: '/material/react-pagination' },
+          { pathname: '/material/react-speed-dial' },
+          { pathname: '/material/react-steppers', title: 'Stepper' },
+          { pathname: '/material/react-tabs' },
+        ],
+      },
+      {
+        pathname: '/material/components/layout',
+        subheader: 'layout',
+        children: [
+          { pathname: '/material/react-box' },
+          { pathname: '/material/react-container' },
+          { pathname: '/material/react-grid' },
+          { pathname: '/material/react-stack' },
+          { pathname: '/material/react-image-list' },
+          { pathname: '/material/react-hidden' },
+        ],
+      },
+      {
+        pathname: '/material/components/utils',
+        subheader: 'utils',
+        children: [
+          { pathname: '/material/react-click-away-listener' },
+          { pathname: '/material/react-css-baseline', title: 'CSS Baseline' },
+          { pathname: '/material/react-modal' },
+          { pathname: '/material/react-no-ssr', title: 'No SSR' },
+          { pathname: '/material/react-popover' },
+          { pathname: '/material/react-popper' },
+          { pathname: '/material/react-portal' },
+          { pathname: '/material/react-textarea-autosize' },
+          { pathname: '/material/react-transitions' },
+          { pathname: '/material/react-use-media-query', title: 'useMediaQuery' },
+        ],
+      },
+      {
```

Comment:
Why are the data-grid pages here?

Reply:
We currently have data-grid in the components. I group all data-grid related to one link so that the people who are familiar with the previous layout don't confuse.

![image](https://user-images.githubusercontent.com/18292247/148023879-51f3e0d9-d12c-4883-a27e-5db2d9651c6e.png)